x-pack/plugins/reporting/public/plugin.ts

```diff
@@ -225,20 +233,23 @@ export class ReportingPublicPlugin
       new ReportingCsvPanelAction({ core, apiClient, startServices$, usesUiCapabilities })
     );
 
-    const reportingStart = this.getContract(core);
+    const reportingStart = this.getContract(shareSetup, core);
     const { toasts } = core.notifications;
 
-    startServices$.subscribe(([{ application }, { licensing }]) => {
+    startServices$.subscribe(([core, { licensing }]) => {
```

Comment:
The `core` name conflicts with a variable already in scope here:
```suggestion
    startServices$.subscribe(([coreStart, { licensing }]) => {
```